### DISCUSSION TITLE
post/chinaDevCommunityProblems/

### DISCUSSION THREAD

**Discussion by giscus** [WhatDamon/whatdamon.github.io#5] [General]
# post/chinaDevCommunityProblems/

免责声明 爱国及立场问题：本人只是为了用较为客观的内容陈述现实，如果国家被污蔑，本人也会打抱不平，在这方面，本人立场与常人一致，即便本文存在很多吐槽，可能还触及到了很大一部分人的敏感部分。  前情提要注意！本文发布时相关事件已经平息，即便依旧有人为此头疼！ 前段时间，很多开发者都发现自己的 GitHub 项目都无缘无故出现在了 GitCode 这个从未听说过的平台，这个平台笔者很早就有所耳闻，毕竟

https://damon233.js.org/post/chinaDevCommunityProblems/

**Comment by ChouChiu** [comment]
捉蟲，需要不是需呀
